haproxy.cfg

```diff
@@ -26,7 +26,6 @@ defaults
 
 frontend dummy-fe
     bind *:80
-    bind *:443
```

Comment:
This file is used only when there are no services. HAProxy does not start without a valid config so I put this dummy file.